src/game/camera.c

```diff
@@ -2627,8 +2637,10 @@ void mode_c_up_camera(struct Camera *c) {
     }
     sPanDistance = 0.f;
 
+    struct Controller* controller = get_mario_controller();
+
     // Exit C-Up mode
-    if (gPlayer1Controller->buttonPressed & (A_BUTTON | B_BUTTON | D_CBUTTONS | L_CBUTTONS | R_CBUTTONS)) {
```

Comment:
im really not a fan of this approach given how most people are used to using gPlayer1Controller. if anything gPlayer1Controller should be turned into a define and made to always point to mario's controller to make sure these code changes don't have to be made